iml-wire-types/src/lib.rs

```diff
@@ -1509,5 +1511,33 @@ impl PartialEq for OstPool {
     }
 }
 
+#[derive(Debug, Clone, serde::Serialize, serde::Deserialize)]
+pub enum Branding {
+    Whamcloud,
+    Ddn,
+    DdnAi400,
```

Comment:
Is an enum really needed? Could it be a plain string? We can style it on display.

Reply:
Personally I think the enum makes sense.

Reply:
I don't :) First of all you are limiting possible branding, adding new will be difficult, will require recompilation, etc. Then, you are basically reading a string, convert it to a enum and back to a string. Why not omit intermediate representation?

Reply:
Because the type system checks the values. What if someone creates a config file and specifies:
"BRANDING=ddm"

Without the enum, it will simply read it as `ddm` and then they will spend time trying to figure out why it's loading the whamcloud logo. By using the enum, it will immediately fail giving an error message that it couldn't parse the branding, allowing you to fix the problem much faster. While it may seem like more boilerplate, I would much rather use the types. If a new logo is added in the future, it still has to be updated in the code anyway.

Reply:
Brandings are very much hardcoded right now. If we get to the point where we allow for any arbitrary branding (loading svg files from outside the binary instead of ones compiled in) then we should revisit this.